ansible_collections/arista/avd/roles/eos_designs/doc/core-interfaces-BETA.md

```diff
@@ -57,6 +57,16 @@ core_interfaces:
       isis_authentication_mode: md5
       isis_authentication_key: $1c$sTNAlR6rKSw=
 
+      # OSPF Parameters
+      ospf_authentication: <1.2.3.4>
+      # OSPF Auth. Parameters
+      ospf_authentication: < simple | message-digest | default -> simple >
```

Comment:
the default here will be the global variable for ospf_authentication before being `simple`. You can use a syntax like in: https://github.com/aristanetworks/ansible-avd/blob/devel/ansible_collections/arista/avd/roles/eos_designs/doc/fabric-topology.md?plain=1#L149